src/view/components/entity-editor/index.js

```diff
@@ -148,6 +152,31 @@ export default class EntityEditor {
     }
   }
 
+  isMissingFields(data) {
+    switch (this.type) {
+      case 'area':
+        return data._id.length === 0 ||
```

Comment:
I'd like to make this definable or expandable in a config of some kind for custom required fields, but at least pulling it into its own method makes it easier to customize in a branch/fork.